clang/test/CIR/Lowering/array.cpp

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o - 2>&1 | FileCheck %s
+
+int a[10];
+// CHECK: @a = external dso_local global [10 x i32]
+
+int aa[10][5];
+// CHECK: @aa = external dso_local global [10 x [5 x i32]]
+
+extern int b[10];
+// CHECK: @b = external dso_local global [10 x i32]
+
+extern int bb[10][5];
+// CHECK: @bb = external dso_local global [10 x [5 x i32]]
+
+void f() {
+  int l[10];
+}
+// CHECK: alloca [10 x i32], i64 1, align 16
```

Comment:
I think it's a good practice to have a check for the function name to establish that we're seeing the `alloca` in the correct context.